EnglishFilter/sections/antiadblock.txt

```diff
@@ -724,6 +724,10 @@ socialgirls.im#%#//scriptlet("prevent-setTimeout", "adblock")
 ! https://github.com/AdguardTeam/AdguardFilters/issues/123928
 @@||bitssurf.com/assets/ads-*.js$~third-party
 bitssurf.com#%#//scriptlet("abort-current-inline-script", "document.getElementById", "AdBlock")
+! https://github.com/AdguardTeam/AdguardFilters/issues/124032
+madshiba.fun#%#//scriptlet('prevent-eval-if', 'mdp-deblocker')
```

Comment:
We have common rules for such detection `! 'mdpDeBlocker'("-blackout active")`.
This rule works fine on my end. Please add the domain to the common rules.
```
madshiba.fun#%#//scriptlet('prevent-eval-if', 'adsBlocked')
```